From de4f941a5d3ff7c04edfe017c0fc6b3373fa78ba Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:05:00 -0400
Subject: [redhat] kabi: Adding symbol node_data (arch/parisc/kernel/parisc_ksyms.c)

Message-id: <d3106592c7f0946560f7d9a06334c09b8d142b22.1412771859.git.poros@redhat.com>
Patchwork-id: 97191
O-Subject: [RHEL7.1 PATCH 40/67] kabi: Adding symbol node_data (arch/parisc/kernel/parisc_ksyms.c)
Bugzilla: 1148030
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148030
    https://bugzilla.redhat.com/show_bug.cgi?id=1148030

    Description:
    ============
    This is standard for working with NUMA hardware like UV,
    it is important for performance to know the topology and
    latency information.
    Symbol used in gru.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/node_data
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/node_data

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/node_data b/redhat/kabi/kabi-module/kabi_ppc64/node_data
new file mode 100644
index 0000000..44f6c1c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/node_data
@@ -0,0 +1,2 @@
+#1-
+0xa91aa598 node_data vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/node_data b/redhat/kabi/kabi-module/kabi_x86_64/node_data
new file mode 100644
index 0000000..1b61e2a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/node_data
@@ -0,0 +1,2 @@
+#1-
+0x3142b10b node_data vmlinux EXPORT_SYMBOL
-- 
1.7.1